src/Model/Indexing/IndexFieldTypes.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Model\Indexing;
+
+final class IndexFieldTypes
+{
+    // ISO 8601 date "2004-02-12T15:19:21+00:00"
+    public const DATEFORMAT = 'c';
+
+    // One would think one could use the named ISO formats (but, NO) and Elastic say they use the format in the link.
```

Comment:
Which format do Elastic say they actually use? https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT? The provided link points to broad information on how to format dates in Java.

Reply:
The default date format in ES do not include time and there predefined name for date-time contains 3 "s" for seconds. Tried to use it but it did not work with the current version of ES... found a lot of folks online complaining about the build in data formats. So ended up using the date time without the extra "s" in the format and ES accepted that format.